model/task/task.go

```diff
@@ -2803,7 +2803,8 @@ func GetAllDependencies(taskIDs []string, taskMap map[string]*Task) ([]Dependenc
 }
 
 func (t *Task) GetHistoricRuntime() (time.Duration, error) {
-	runtimes, err := getExpectedDurationsForWindow(t.DisplayName, t.Project, t.BuildVariant, t.FinishTime.Add(-oneMonthIsh), t.FinishTime.Add(-time.Second))
+	runtimes, err := getExpectedDurationsForWindow(t.DisplayName, t.Project, t.BuildVariant,
```

Comment:
Code LGTM, so it seems `withStdDev` is false always for now? Do we have a plan to change this in the future if we see some certain behavior once this is deployed?

Reply:
Oh that was a mistake; this should be true for FetchExpectedDuration since that records the std dev. I'll update.